site/ko/guide/eager.ipynb

```diff
@@ -0,0 +1,1237 @@
+    "id": "rnZXjfQzqEhV"
+   },
+   "source": [
+    "## 성능\n",
+    "\n",
+    "즉시 실행에서 계산은 자동으로 GPU로 분배됩니다.\n",
+    "만약 계산 분배를 사용자가 제어하고 싶다면 그 부분을 `tf.device('/gpu:0')` 블록 (또는 CPU도 동일)으로 감싸서 실행하세요:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "attributes": {
+     "classes": [
+      "py"
+     ],
+     "id": ""
+    },
+    "colab": {},
+    "colab_type": "code",
+    "id": "Ac9Y64H-qEhX"
+   },
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "def measure(x, steps):\n",
+    "  # 텐서플로는 먼저 사용했던 GPU를 초기화, 시간계산에서 제외\n",
+    "  tf.matmul(x, x)\n",
+    "  start = time.time()\n",
+    "  for i in range(steps):\n",
+    "    x = tf.matmul(x, x)\n",
+    "  # tf.matmul는 행렬 곱셈을 완료하기 전에 결과를 반환한다\n",
+    "  # (예, CUDA 스트림 대기열에 연산을 추가한 후에 결과를 반환할 수 있다).\n",
+    "  # 아래 x.numpy() 호출은 대기열에 추가된 모든 연산이 완료될 것임을 보장할 것이다\n",
+    "  # (그리고 그 결과가 호스트 메모리에 복사될 것이고,\n",
+    "  # 그래서 matnul 연산시간보다는 조금 많은 연산시간이\n",
+    "  # 포함될 것이다).\n",
+    "  _ = x.numpy()\n",
+    "  end = time.time()\n",
+    "  return end - start\n",
+    "\n",
+    "shape = (1000, 1000)\n",
+    "steps = 200\n",
+    "print(\"Time to multiply a {} matrix by itself {} times:\".format(shape, steps))\n",
+    "\n",
+    "# CPU에서 실행:\n",
+    "with tf.device(\"/cpu:0\"):\n",
+    "  print(\"CPU: {} secs\".format(measure(tf.random.normal(shape), steps)))\n",
+    "\n",
+    "# GPU에서 실행, 가능하다면:\n",
+    "if tf.config.experimental.list_physical_devices(\"GPU\"):\n",
+    "  with tf.device(\"/gpu:0\"):\n",
+    "    print(\"GPU: {} secs\".format(measure(tf.random.normal(shape), steps)))\n",
+    "else:\n",
+    "  print(\"GPU: not found\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "RLw3IS7UqEhe"
+   },
+   "source": [
+    "`tf.Tensor` 객체는 실제로 그 연산을 수행할 다른 디바이스로 복사될 수 있습니다:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "attributes": {
+     "classes": [
+      "py"
+     ],
+     "id": ""
+    },
+    "colab": {},
+    "colab_type": "code",
+    "id": "ny6LX2BVqEhf"
+   },
+   "outputs": [],
+   "source": [
+    "if tf.config.experimental.list_physical_devices(\"GPU\"):\n",
+    "  x = tf.random.normal([10, 10])\n",
+    "\n",
+    "  x_gpu0 = x.gpu()\n",
+    "  x_cpu = x.cpu()\n",
+    "\n",
+    "  _ = tf.matmul(x_cpu, x_cpu)    # CPU에서 실행\n",
+    "  _ = tf.matmul(x_gpu0, x_gpu0)  # GPU:0에서 실행"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "oA_qaII3-p6c"
+   },
+   "source": [
+    "### 벤치마크\n",
+    "\n",
+    "GPU에서 학습을 하는 [ResNet50](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/eager/python/examples/resnet50) 같은 계산량이 많은 모델에서,\n",
+    "즉시 실행 성능은 `tf.function` 실행과 비교될 수 있습니다.\n",
+    "그러나 이러한 차이는 계산량이 작은 모델인 경우 더 커지고, 수많은 작은 연산으로 구성된 모델은 자주 반복되는 부분을 최적화하는 사례도 있습니다.\n",
+    "\n",
+    "## 함수(functions)를 활용\n",
```

Comment:
In TF 2.0, 'functions' have not specific meaning.
I suggest to remove '(functions)'.

Reply:
Thanks, fixed